templates/zerver/register.html

```diff
@@ -153,22 +153,16 @@
           <div class="help-box">
             <div id="org_type_blob_1" class="blob display-none">
               {% trans %}
-              Create a corporate organization if your users will be members of the
-              same company, where the privacy expectation is that the company's
-              organization administrators may need to have access to your users'
-              private message history. Zulip features that allow organization
-              administrators to take control of accounts or access private message
-              history for other users are only available to corporate organizations.
+              Organization admins will be able to take control of users'
+              accounts and access their private messages. Good for corporate
```

Comment:
Most corporations do need such features, for better or worse. Often for compliance or other legal reasons. This is actually the default for most products; e.g. the admin of a slack team can take control of user accounts and access their private messages (probably not on their free tier, but I expect at some tier).

I agree this section needs some thought; let's leave it as is for now, and then figure out the exact thing to do when Tim gets back.